lib/asciidoctor-pdf/converter.rb

```diff
@@ -210,6 +210,12 @@ def convert_document doc
     font @theme.base_font_family, size: @theme.base_font_size, style: @theme.base_font_style.to_sym
     doc.set_attr 'pdf-anchor', (doc_anchor = derive_anchor_from_id doc.id, 'top')
     add_dest_for_block doc, doc_anchor
+    unless doc.doctype == 'book' || (doc.attr? 'title-page')
```

Comment:
Adding ` || doc.notitle` will solve the issue #998 

    unless doc.doctype == 'book' || (doc.attr? 'title-page') || doc.notitle

Did it on my fork, works for me (not sure how to properly separate stuff, waiting for my PR to be included/rejected by you and then I will probably be able to properly use feature branches)

Reply:
This will be handled by the fix for #998. I've proposed a PR.